### ISSUE TITLE
Review use of mandatory fields

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#1915]
Mandatory fields were not required for census, however GDS advised that mandatory and non-mandatory fields should be highlighted. Spike required to consider how this should be applied going forward

**Comment by jrbarnes9** [comment]
https://govuk-elements.herokuapp.com/form-elements/#form-optional-fields

**Comment by daniellecorke** [comment]
See 'When to use this pattern': [Gov Design System Question Pages](https://design-system.service.gov.uk/patterns/question-pages/ )

**Comment by daniellecorke** [comment]
The eQ DAC test highlighted that not providing optionality was a Level A fail therefore it would be good to prioritise this work. eQ are planning for a DAC re-test in June.

**Comment by daniellecorke** [comment]
**eQ DAC suggested solution:**
Ensure that where both required and optional fields are used, that either the required or the optional fields are identified as such to users. Ensure that this is implemented consistently throughout the service.

**Comment by daniellecorke** [comment]
On the UCD Board: https://jira.ons.gov.uk/browse/CU-97.

**Comment by e-livingstone** [comment]
This is in the UCD prioritised backlog
